Jenkinsfile

```diff
@@ -13,43 +13,65 @@ pipeline {
         staticImageARM32 = ''
         staticImageARM64 = ''
     }
+
     stages {
-        stage('Build & test') {
+
+        stage('Git') {
             agent {
                 docker {
                     image 'teambitflow/golang-build:debian'
                     args '-v /tmp/go-mod-cache/debian:/go'
-                }
+                    }
             }
-            stages {
-                stage('Git') {
-                    steps {
-                        script {
-                            env.GIT_COMMITTER_EMAIL = sh(
+                    
+            steps {
+                script {
+                    env.GIT_COMMITTER_EMAIL = sh(
                                 script: "git --no-pager show -s --format='%ae'",
                                 returnStdout: true
                                 ).trim()
+                }
+            }
+        }
+
+        stage('Build & test') {
+
+            parallel {
+
+                stage('Unit tests') {
+                    agent {
+                        docker {
+                            image 'teambitflow/golang-build:debian'
+                            args '-v /tmp/go-mod-cache/debian:/go'
                         }
                     }
-                }
-                stage('Build & test') {
+
                     steps {
-                            sh 'go clean -i -v ./...'
-                            sh 'go install -v ./...'
-                            sh 'rm -rf reports && mkdir -p reports'
-                            sh 'stdbuf -o 0 go test -v ./... -coverprofile=reports/test-coverage.txt | tee reports/test-output.txt' // Use stdbuf to disable buffering of the tee output
-                            sh 'cat reports/test-output.txt | go-junit-report -set-exit-code > reports/test.xml'
-                            sh 'go vet ./... &> reports/vet.txt'
-                            sh 'golint $(go list -f "{{.Dir}}" ./...) &> reports/lint.txt'
-                    }
+                        sh 'go clean -i -v ./...'
+                        sh 'go install -v ./...'
+                        sh 'rm -rf reports && mkdir -p reports'
+                        sh 'stdbuf -o 0 go test -v ./... -coverprofile=reports/test-coverage.txt | tee reports/test-output.txt' // Use stdbuf to disable buffering of the tee output
+                        sh 'cat reports/test-output.txt | go-junit-report -set-exit-code > reports/test.xml'
+                        sh 'go vet ./... &> reports/vet.txt'
+                        sh 'golint $(go list -f "{{.Dir}}" ./...) &> reports/lint.txt'
+                    } 
+
                     post {
                         always {
                             archiveArtifacts 'reports/*'
                             junit 'reports/test.xml'
                         }
                     }
                 }
+
                 stage('SonarQube') {
+                    agent {
```

Comment:
SonarQube actually uses the vet/lint/test results from the Unit test stage, so I think at least these two cannot be parallelized

Reply:
Fixed that.

Should I change the DockerHub org to "bitflowstream"?